.travis.yml

```diff
@@ -26,9 +26,7 @@ before_script:
   - git show --pretty=fuller -s
   - mkdir -p coverage
   - etc/install_julia_gap.sh
```

Comment:
Should I try to put #186 into this PR? I am not 100% sure, but it might need adjustments

Reply:
Well, currently, I still build against the extra compiled GAP. I can of course only use the GAP installed by julia

Reply:
I am confused; are you saying you are downloading and building GAP (and all packages, etc.) twice now?!

Reply:
No. We just do not download the GAP via Julia, but use the GAP installed by our script.

This way, it is easier to install `profiling` and `io` afterwards.

Reply:
I don't understand. Why is it easier to install profiling and io this way, resp. what's the problem with installing them in the GAP installed by Julia? Because you don't know the directory? Well, seeing that we already have the `GAP_INSTALL_PACKAGES`, couldn't it also have a "debug" mode where the `build.jl` installs io and profiling?

Also, does this mean building the Julia package is not actually tested? It seems to me that we should have one Travis config using our own GAP, and one using the GAP installed via Julia, to make sure both variants keep working... That said, it is fine to implement this *after* this PR has been merged; I am mostly bringing it up here to understand what the issues are you are working around here.

Reply:
Exactly the fact that we do not know in which directory the GAP that Julia compiles is compiled to. I guess the easiest is the debug flag you propose. Should we do that?